### DISCUSSION TITLE
Make dark state selectable in config flow

### DISCUSSION THREAD

**Discussion by klatka** [jseidl/magic-areas#279] [Ideas / Feature Requests]
Currently only overhead lights are turned on if the area is occupied and extended state switches from bright to dark.

In the config flow the dark state is not selectable for different light groups so there is no way for the user to change it.
For example, if I sit in my workroom and the area becomes dark because my sensor reports low illuminance, I want to get my task lights turned on instead of the overhead lights.

Sleep and accented modes are selectable. Is there any reason why dark is excluded from that list (see [config_flow.py](https://github.com/jseidl/hass-magic_areas/blob/master/custom_components/magic_areas/config_flow.py#:~:text=if%20extra_state%20in%20LIGHT_GROUP_STATE_EXEMPT%3A))?

**Comment by jseidl** [comment]
The Light groups (overhead, task, sleep and accent) are not tied to any state. By default the overhead lights are tied to the `occupied` state but you can remove from there and add in other groups such as the task light if you want.

Dark state is not selectable because the lights wont turn on if it's not dark so there's no reason to select it anyway. What you seem to be looking for is the `occupied` state.
